src/public/modules/forms/helpers/CsvMergedHeadersGenerator.ts

```diff
@@ -109,26 +114,44 @@ class CsvMergedHeadersGenerator extends CsvGenerator {
     // Craft a new csv row for each unprocessed record
     // O(qn), where q = number of unique questions, n = number of submissions.
     this.unprocessed.forEach((up) => {
-      let createdAt = moment(up.created).tz('Asia/Singapore')
-      createdAt = createdAt.isValid()
+      const createdAt = moment(up.created).tz('Asia/Singapore')
+      const formattedDate = createdAt.isValid()
         ? createdAt.format('DD MMM YYYY hh:mm:ss A')
-        : createdAt
-      let row = [up.submissionId, createdAt]
-      for (let [fieldId] of this.fieldIdToQuestion) {
+        : createdAt.toString() // just convert to string if given date is not valid
+      const row = [up.submissionId, formattedDate]
+
+      this.fieldIdToQuestion.forEach((_question, fieldId) => {
         const numCols = this.fieldIdToNumCols[fieldId]
         for (let colIndex = 0; colIndex < numCols; colIndex++) {
           row.push(this._extractAnswer(up.record, fieldId, colIndex))
         }
-      }
+      })
       this.addLine(row)
     })
     this.hasBeenProcessed = true
   }
 
+  /**
+   * Extracts the string representation from a field response
+   * @param unprocessedRecord
+   * @param fieldId
+   * @param colIndex
+   * @returns string representation of unprocessed record
+   */
+  _extractAnswer(
```

Comment:
missed this, but if this is private we can use the `#` modifier too

Reply:
We can't do this because of the ECMA version :(
```
ERROR in /home/travis/build/opengovsg/FormSG/src/public/modules/forms/helpers/csv-response-classes/SingleAnswerResponse.class.ts(6,3)
          TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.
```

Reply:
Somehow my comment didn't go through, our TSConfig is indeed targeting ES6 though... but probably not worth it to debug. 

We can use Typescript's own [`private`](https://www.typescriptlang.org/docs/handbook/2/classes.html#private) modifier in that case